src/routes/docs/products/auth/mfa/+page.markdoc

```diff
@@ -0,0 +1,167 @@
+---
+layout: article
+title: Multi-factor authentication
+description: Add multiple layers of authentication to your applications powered by Appwrite Authentication.
+---
+
+Multi-factor authentication (MFA) greatly increases the security of your apps by adding additional layers of protection.
+When Multifactor authentication is enabled, a malicious actor needs to compromise multiple authentication factors to gain unauthorized access.
+Appwrite Authentication lets you easily implement MFA in your apps, letting you build more securely and quickly.
+
+Appwrite currently allows two factors of authentication. More factors of authentication will be available soon.
+
+Here are the steps to implement MFA in your application.
+
+{% section #enable-mfa step=1 title="Enable MFA on an account" %}
+[TODO @TorstenDittmann need some advice here, what's the steps?]
```

Comment:
@TorstenDittmann this is the MFA journey docs we added. Please take a look. Need help double checking the flow + the missing auth methods :)